Assets/RGLUnityPlugin/Scripts/SceneManager.cs

```diff
@@ -47,6 +47,9 @@ public enum MeshSource
         [SerializeField]
         private MeshSource meshSource = MeshSource.RegularMeshesAndSkinnedMeshes;
 
+        [SerializeField]
+        private bool textureReading = false;
```

Comment:
It also turns off gathering texture coordinates from meshes. 
Saves a lot of data transfer.
Consider this again.

Reply:
> If we will leave it we can also turn off gathering texture coordinates from meshes.
Saves a lot of data transfer.

It could be achieved by adding a public function to the RGLMesh that uploads UVs if not uploaded. That function will be called in the `SetIntensityTexture()`. My proposal is even better because it skips UVs uploading if the entity hasn't intensity texture.